review/fileToReview.js

```diff
@@ -0,0 +1,73 @@
+
+var domain = "bank.local.fr"
+
+/**
+ * @description Fetch transactions recursively
+ * @param {string} fromDate The maximum date of transactions to return
+ * @param {string} authorization Authorization header to authent the user
+ * @param {jws} jws Jws token, mandatory if we get one from login request
+ * @param {Number} id Account id
+ * @param {Number} page Number of the page
+ * @param {Object} previousTransactions Previous page of transactions (To ckeck for dupes)
+ * @return {Object} All transactions available on the page
+ */
+async function fetchTransactions(fromDate, authorization, jws = null, id, page, previousTransactions) {
+	console.log(`--- Fetch Trasactions page n°${page} ---`);
+	try {
+    var headers = {"Authorisation":  authorization }
+
+    if (jws) {
+      headers = {
+        "Authorisation": authorization,
+        "jws": jws,
+        "Content-type": "application/json",
+        "Accept": "application/json"
+      }
+    } else {
+      headers = {
+        "Authorisation": authorization,
+        "Content-type": "application/json",
+        "Accept": "application/json",
+      }
+    }
+
+	  var {code, response } = await doRequest('GET',
+      domain + '/accounts/'+ id + '/transactions?' + `page=${page}`,
+      headers);
```

Comment:
Where is `doRequest` method define of imported?
It's way more clean if we avoid adding string but use **`**  instead 
```suggestion
	  const { code, response } = await doRequest('GET', `${domain}/accounts/${id}/transactions?page=${page}`, headers);
```